openseadragon-scalebar.js

```diff
@@ -371,6 +371,22 @@
             var vCenter = canvas.height / 2;
             context.fillText(this.divElt.textContent, hCenter, vCenter);
             return canvas;
+        },
+        /**
+         * Inject scalebar into a canvas with viewport image.
+         * @returns {Element} A canvas containing the scalebar representation
+         */
+        injectIntoImageCanvas: function() {
```

Comment:
I think that `getImageWithScalebarAsCanvas` would be a better name. Would you mind changing it?

For the documentation, I suggest:
>Get a copy of the current OpenSeadragon canvas with the scalebar.
>@returns {Element} A canvas containing a copy of the current OpenSeadragon canvas with the scalebar